src/aics-image-viewer/assets/styles/variables.css

```diff
@@ -3,7 +3,7 @@
 /* $app-height: 100%; */
 /* $dark-gray: #313131; */
 /* $purple: #8048f3; */
-$highlight-purple: #9e6aff;
+$highlight-purple: #aa88ed;
```

Comment:
Kind of a cheat here. Highlight purple was being used for button hovering, so I replaced it with the new light purple from the palette. This eventually needs to be replaced.